File: app/src/main/java/com/jinjoo/pouch/model/pub/Res.java
package com.jinjoo.pouch.model.pub;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root
public class Res {
    @Element
    Header header;

    @Element
    Body body;


    public Header getHeader() {
        return header;
    }

    public void setHeader(Header header) {
        this.header = header;
    }

    public Body getBody() {
        return body;
    }

    public void setBody(Body body) {
        this.body = body;
    }

    @Override
    public String toString() {
        return "Res{" +
                "header=" + header +
                ", body=" + body +
                '}';
    }
}

File: app/src/main/kotlin/com/jinjoo/pouch/api/model/pub/Res.kt
package com.jinjoo.pouch.api.model.pub

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root


@Root(name = "response", strict = false)
data class Res (
        @field:Element(name = "header") var header: Header? = null,

        @field:Element(name = "body") var body: Body? = null
)